packages/squiggle-lang/src/rescript/Reducer/Reducer_Expression/Reducer_Expression.res

```diff
@@ -156,7 +107,7 @@ let reduceExpression = (expression: t, bindings: T.bindings): result<expressionV
         )
         racc->Result.flatMap(acc => acc->reduceValueList)
       }
-    | T.EBindings(_) => RETodo("Cannot return bindings")->Error
+    | EBindings(_bindings) => RETodo("Error: Bindings cannot be reduced to values")->Error
```

Comment:
Same comment as above (I may be wrong). I mostly want to silence warnings.

Reply:
This doesn't create a warning because it starts with an underscore. I chose this implementation because I like still having the name

Reply:
:+1: